ext/dep_gecode/dep_selector_to_gecode.cpp

```diff
@@ -633,6 +649,8 @@ VersionProblem * VersionProblem::InnerSolve(VersionProblem * problem, int &iterc
                     delete best_solution;
                 }
             best_solution = solution;
+            best_solution->solutionState = SOLUTION_STATE_SOLVED;
```

Comment:
Is solutionState not set by the solver correctly?  Not clear why we have to explicitly set it here..